src/DoctrineORMModule/Module.php

```diff
@@ -142,5 +143,8 @@ public function initializeConsole(EventInterface $event)
         $helperSet->set(new QuestionHelper(), 'question');
         $helperSet->set(new ConnectionHelper($entityManager->getConnection()), 'db');
         $helperSet->set(new EntityManagerHelper($entityManager), 'em');
+        if (class_exists('Doctrine\\DBAL\\Migrations\\Version')) {
+            $helperSet->set(new DialogHelper(), 'dialog');
```

Comment:
I don't know.. For me this is a good idea.. But I can move this dependency how required
https://github.com/doctrine/DoctrineORMModule/blob/master/composer.json#L49

:) Now is require dev.. For me it's a good stuff